domain/include/cstone/halos/halos.hpp

```diff
@@ -173,7 +171,7 @@ class Halos
                                   << " of similar magnitude than the rank domain volume. In that case, either"
                                   << " the number of ranks needs to be decreased or the number of particles"
                                   << " increased, leading to shorter smoothing lengths\n";
-                        MPI_Abort(MPI_COMM_WORLD, 1);
+                        MPI_Abort(MPI_COMM_WORLD, 35);
```

Comment:
Is this MPI_Abort to catch some lost rank? 
Is 35 a random value, or only to differentiate the exit value to something else?

Reply:
I observed this MPI_Abort statement being triggered without getting the error print message above. That's why I'm
giving a unique return value (35) here to faster determine where the Abort came from, even if the error is not printed.

Reply:
ok, thanks